portal/exercise_diet/templates/exercise_diet/layout.html

```diff
@@ -3,22 +3,22 @@
 <head>
     <meta charset="UTF-8">
     
-    <title>Exercise Diet</title>
+    <title>{% trans %}Exercise Diet{% endtrans %}</title>
 
-    <script src="https://code.jquery.com/jquery-1.12.1.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/js/bootstrap.min.js"></script>
     
-    <link rel="stylesheet" type="text/css" href="https://demo-st.us.truenth.org/css/bootstrap.css" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/css/bootstrap.css" rel="stylesheet">
 <!--     <link rel="stylesheet" type="text/css" href="https://demo.us.truenth.org/gil/static/css/gil.css" rel="stylesheet"> -->
     <link rel="stylesheet" type="text/css" href="https://demo-st.us.truenth.org/css/cpro.css">
-    <link rel="stylesheet" type="text/css" href="//demo.us.truenth.org/static/css/topnav.css" rel="stylesheet" />
-    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-    <link rel="stylesheet" type="text/css" href="../static/css/styles.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.rawgit.com/uwcirg/true_nth_usa_portal/develop/portal/static/css/topnav.css" rel="stylesheet" />
```

Comment:
Todo: replace with `url_for()` referencing `topnav.css` from main blueprint

Reply:
that won't work in the context of CHCR's standalone.  we'll patch that after final handoff.